pkg/kubelet/dockertools/manager_test.go

```diff
@@ -30,7 +30,6 @@ import (
 	"time"
 
 	dockertypes "github.com/docker/engine-api/types"
-	containertypes "github.com/docker/engine-api/types/container"
```

Comment:
weird that it's not an error or warning to import the same thing two different ways.